docs/source/example_notebooks/prediction/dowhy_causal_prediction_demo.ipynb

```diff
@@ -0,0 +1,1526 @@
MUJYNkJSU1VvczklMkJoRTh1cG8lMkZmMDFnWHdsd2cwdCUyQlJpYkl6bEE1Z1FwdURFZyUyQjlhc29MY3hsT3ljZ0lxQ1glMkJlVW5qblRpcEdmYzdXVHFmYkxSRnRRcmJEUzk3MTRGNEI0WmVXMG1UVFptSnE5YjJyZCUyRk02N2oxbmd2bjB6JTJCUmFoeXl5YnpKSW5vTGxZQXNQeXJDdURnUFY0dk5QUVBST1hCdFA4bUlhWGI2WjVPT3dPOExCemZ4a2FEN0R5dlJmdVljeHBtWlVUUyUyQmJaMWl3RHl5eXV4QWVRJTJCRnJMb0laRmhxUFhwek1CMWdaS3Q1cXVQVGFmR2JSWmx6TzNIUXMzSnZBaEI2ZElxJTJGQTNFMHMlMkZaUUtVMGgzT1FNdnVHZDFaMWVPM2VsSUs2U0RQU2M5Z0clMkZYNkZJQ2JLYXpkdzVJdlVCJTJGTk1lS2c3dk9wZnlUT2p4cmVqZUt6M2JIcE50b3BpNlZVdnJaOWU5cSUyQkdMMG50JTJGT0licWhwRkdwOFBqdlNNY0F2JTJCUlBHOWp1MUI5ZFE5ODBuVExSVXRqWHFETG5aOXp5JTJGVWtvcU5LamklMkJBekglMkZCT0ZYemNURlZCWW5nSHBMMDg2aDRERWdzVlFVMHlYeDN1Vjhpc2U3UFFWWUlpQTVIWUEzanVBNDIlMkJjdjdBblRWSVFEUCUyRlhIZWl2JTJCb3cyTEFTdVolMkJEMERZaFJsbFJJQmc3aHh2RHZTU3NtcXVtRUZJdjh0RFRZS0klMkJmUml4STl0S2JZMElLSnhOUlFzWVZHRkRCZU5KUU5MMUhjeUtaeFpsSTdGeGVVSG0lMkZCU2xxbVVObU5VRzh1RnZ1elkzUmdDZXNYciUyRmh0QXBQQnJGbDNEU0hWMGxQbG15aFQ2ZXFCJTJGeUpFajB3SUExUk9iRU92V21KdFBndnZGcnoyeHdleHc0dWtibEJnUDlEayUyQm42RnphMVd2MTRhcGhaSnpIYWllc2FyR2xyRGo0RCUyRnVwb0NaeE84bkI5JTJCTGh4amtiR3FYOVdpU0QlMkZwZUJWNVZoRzRVRFk4b2d4VnJDM1hrZ2Fic21JUWpvbzNLV3ZuTEwlMkI4dkRqYXJMS2RLbnkxWndiN3drdThmWVk4Nkh0TnFjQmlXYjZQYVpLajhoOGVLZSUyRnh2cjhNcjdTRmYlMkZZeGlSbmhDZmsxVDdKWEFOd1glMkJyQiUyQkV1ZTlmT09QVGxVdk1uejFlJTJCODFhY2FwSzdPdlo0V0Ewa3h0UzVMNXNqa2V5WkpxQmwlMkJUU0YlMkZQOHh2OFk0N09FZGMlMkJkd3prd0piUWRVVTV2aGFpemRFQ2lKdXdWWGdtM0lHbFhjMHRBc1JSMFBxWlhIelRXd1BIYWQyb1VrTWVvN3RhcGZkOW9oNVYlMkZSMTJtT3BxJTJCeENLOVlzUGNSZU4lMkZTV3hoZGJmYmhGN2dYJTJGZTFoZWJTNnp1WTJmamZQSEozd0FPNjVGY0xlVjQ2QW1HMCUyQm1XWFBCJTJGT3c3Um5yNGs0WVF2Z3ZGTnUlMkZzYlNySyUyQnZGUE51cFZrdUZRTkZFZHlRRHBXVWtpNEtmejRZb0ozMWYyMTJwMVNRb2lYYmRyWjN1a3hmaFZhYVRsMXljTjBQMGhHdDJPcyUyQnd0RDBRRXJKOWVPc042UldIbyUyQmElMkJBcm0yM2RzY1c1Q2U2ZUM5dUtFZVp6dnB6eWlJWUVJbUtnMTd2MGZQOTNic2E0WDhxSjhhU2VuNnpEekFvanduMjN4biUyQk9LNmZSM3lOMkRsRHQlMkJ5Zmc2WlZuQUF2RTJXMm53UGZjJTJGbnNmN3dmJTJGdjhFbzRLellrTG9PbW9UNDJiSzhDYVFLUUpLWXRIMkIyYUllSWxncU5aS2VGajRHUWU1Y2dRd1BsV0NQQmJxOFpMdnZ2d1MxbDM2MnhyOXpiOTJVZVh4cDgxd0h5YVVUVzNKM1BpOExBQkkxMyUyQld1NkZGSCUyQlhxVUpFWHJLOEptZE55eSUyRkFib1ppckI3emtVRmZrVzh5anRzclQ0JTJGdXNhaWolMkZIZVRrdVRtN0YwVFRVQjlYcmFOcGQ4WVJoMXo3aURhZ3NQMldSVlFGQkFKc1VySldKeDQ2aTRDaFVxcGp2M1phODVhU041JTJGaXF6M21FaSUyRms2JTJCTmhFU1Brdk9sWVd3czdKOTZ1OXJVVDFlUk0yV254TkFYc3lEMzJaSyUyRjZLaDVvUXkzN2swWnFHUzlrZ3VVNHNFVzhMNllOMXlmQ3UlMkZHRVQ2SDlueXNudHIzN0liaXpwaFVGS1Z3QVBHMFI2Y011OWRxMmJxOEpzeE56UURvOGElMkZ5eGlSMTFCWmUlMkJZcUZOSjIlMkJPRFFBYkFEMmpkUnFxSyUyQjJWdmx3Rm5CelJxUnpwSzVVc3BMRjFlSmJuclpTSG1LbjYyYzA2MGYxcFFPclEySXBNVlpjMnhZUGpLNUtuWjRvaEZKaGtkOCUyQlRVb1AzT0JGS01YUVhMVGw0bnFicUxQdWRWZXhwY0h0ZVhIQ0g1ZXFjYm9UeERndG8lMkY3dkFUMDNoWW90MmIyVTV4dE4wTWo0UTJFNFlPajI3dEI0eVRMTU5oMFFkekxQMWxYbE05MmhlczZyTjZuUmQlMkYyeDZKb1RXbEhFb2h6NGpjeFNXOUYlMkJWTG10WjczSzhjQ0tmbTNQNDBHTnRWRkVRMHBIN29JOEFtVkpJZmpxQTZETlZXQmoyVTRvdSUyQlN6NEZYZkJwVjIyeHZwNHY0SmZUOHdYcnUweiUyRk84YTh2MUNrWXM3ZGRJejJ2d1I3Uk9hNm8ydXpjdnFtJTJGcEJrN3FEdEklMkJsRlUyZExycGZUTHdKQVpISndIRXFvV3VFb2Vqejlu
```

Comment:
is there a way to avoid this verbose output? can you output a progress bar using tqdm?

Reply:
okay, it is fine then